packages/generator/src/edmx-parser/legacy/common/edmx-to-vdm.ts

```diff
@@ -480,7 +480,7 @@ export function transformFunctionImportBase(
       ),
       edmType: parseType(p.Type),
       jsType: edmToTsType(p.Type)!,
-      fieldType: edmToFieldType(p.Type),
+      fieldType: edmToFieldType(p.Type, true),
```

Comment:
This seems wrong. Why it is always nullable?

Reply:
See above.